sofiadinizms/NFT-onepage
# Initial config+db

## What I did

1. Fixed and added back-end configurations
2. Created the model and route for the basic section layout

## How to test it

1. Install the project dependencies (just to make sure, I recommend using both yarn and npm install) on both the client and server folders
2. Run the back end with `npm start` and access `http://localhost:3001/admin/basic-sections` where you should be able to view, create, edit and delete sections

## Technical debts
1. Backend tests are still missing
2. I have yet to analyze the switch to Firebase
